agent/app/agent.go

```diff
@@ -323,7 +325,7 @@ func (agent *ecsAgent) newTaskEngine(containerChangeEventStream *eventstream.Eve
 	}
 
 	// We try to set these values by loading the existing state file first
-	var previousCluster, previousEC2InstanceID, previousContainerInstanceArn, previousAZ string
+	var previousCluster, previousEC2InstanceID, previousContainerInstanceArn, previousAZ, previousRegistrationToken string
```

Comment:
I do not think we still need previousRegistrationToken here, since we remove registrationToken from state file, it will never be initialized here.